juniper_graphql_ws/Cargo.toml

```diff
@@ -3,17 +3,28 @@ name = "juniper_graphql_ws"
 version = "0.4.0-dev"
 edition = "2021"
 rust-version = "1.65"
-description = "Legacy `graphql-ws` GraphQL over WebSocket Protocol implementation for `juniper` crate."
+description = "GraphQL over WebSocket Protocol implementations for `juniper` crate."
 license = "BSD-2-Clause"
-authors = ["Christopher Brown <ccbrown112@gmail.com>"]
+authors = [
+    "Christopher Brown <ccbrown112@gmail.com>",
+    "Kai Ren <tyranron@gmail.com>",
+]
 documentation = "https://docs.rs/juniper_graphql_ws"
 homepage = "https://github.com/graphql-rust/juniper/tree/master/juniper_graphql_ws"
 repository = "https://github.com/graphql-rust/juniper"
 readme = "README.md"
 categories = ["asynchronous", "web-programming", "web-programming::http-server"]
-keywords = ["apollo", "graphql", "graphql-ws", "subscription", "websocket"]
+keywords = ["apollo", "graphql-transport-ws", "graphql-ws", "subscription", "websocket"]
 exclude = ["/release.toml"]
 
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[features]
+graphql-transport-ws = []
+graphql-ws = []
```

Comment:
Why gate these behind features at all? The modules for these are small and don't pull in additional dependencies. Unless I'm missing something, as a user I'm not sure why I would ever want to explicitly enable or disable these.

Reply:
@ccbrown I was thinking that some people may want to opt-in for the concrete protocol version and don't consider another one at all, thus avoid increasing compilation times and auto-completion pollution in any way.